.github/CODEOWNERS

```diff
@@ -297,6 +297,9 @@ libs/clients/rsk/relationships/
 /libs/application/template-api-modules/src/lib/modules/templates/directorate-of-immigration/             @island-is/origo
 /libs/clients/directorate-of-immigration/                                                                @island-is/origo
 
+/libs/application/templates/healthcare-license-certificate/                                              @island-is/origo
+/libs/application/template-api-modules/src/lib/modules/templates/healthcare-license-certificate/         @island-is/origo
+
```

Comment:
Perhaps you should add `@island-is/origo` as co-owners of `/libs/clients/health-directorate` along with `@island-is/hugsmidjan`. 😃